<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoriesModule</class>
 <widget class="QWidget" name="CategoriesModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="categoriesWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="softwareBox">
         <property name="title">
          <string>Software</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="softwareWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>true</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Fixed</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Update</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/48x48/apps/system-software-update.png</normaloff>:/icons/48x48/apps/system-software-update.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Install</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/48x48/actions/list-add.png</normaloff>:/icons/48x48/actions/list-add.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Package Manager</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/48x48/apps/preferences-desktop-default-applications.png</normaloff>:/icons/48x48/apps/preferences-desktop-default-applications.png</iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
